### DISCUSSION TITLE
Can we receive an email for each submission?

### DISCUSSION THREAD

**Discussion by jill88** [OpnForm/OpnForm#620] [Q&A] [answered]
I've enable the email integration that emails the user when they fill a form.  But I don't see how to enable an email notification to the admin that says a new submission has been received.  Am I missing something?

**Comment by JhumanJ** [comment, marked as answer]
Hey have you checked this article?
https://help.opnform.com/en/article/can-i-receive-notifications-on-form-submissions-134svqv/
